src/uint.rs

```diff
@@ -166,6 +169,37 @@ impl<const LIMBS: usize> fmt::UpperHex for UInt<LIMBS> {
     }
 }
 
+#[cfg(feature = "serde")]
+#[cfg_attr(docsrs, doc(cfg(feature = "serde")))]
+impl<'de, const LIMBS: usize> Deserialize<'de> for UInt<LIMBS>
+where
+    UInt<LIMBS>: Encoding,
+{
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        let mut buffer = Self::ZERO.to_le_bytes();
```

Comment:
Couldn't come up with a cleaner way.

Reply:
Yeah, it's kind of tough. I think `GenericArray::default()` would work if you use `Self::from_le_byte_array` and the `ArrayEncoding` trait.

Otherwise there's `UInt::<Limbs>::Repr::default()` but unfortunately those impls only go up to 32-bytes, so not only would you need a `Default` bound to use them but also it would limit the serializable size to `U256` unnecessarily.